packages/core-mobile/app/services/bridge/BridgeService.ts

```diff
@@ -47,6 +49,7 @@ type TransferEVMParams = {
   onStatusChange?: (status: WrapStatus) => void
   onTxHashChange?: (txHash: string) => void
   request: Request
+  activeNetwork: Network
```

Comment:
```suggestion
  network: Network
```
since we are getting rid of active network soon